Jenkinsfile

```diff
@@ -90,50 +103,24 @@ timestamps {
               archiveArtifacts 'coverage-report/lcov-report/index.html'
             }
           }
-        }
-
-        stage('Sonar') {
-          onPR {
```

Comment:
We are loosing this, anyone to miss it? There was no much value in this so far I we couldn't get sonar plugin to comment on PR's so we were unable to really see any issue.